rpc/common/service.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.34.0
-// 	protoc        v3.19.4
+// 	protoc-gen-go v1.34.2
+// 	protoc        v3.17.3
```

Comment:
How did you generate it? Did you run `mage protoc`?

Reply:
I used a local protoc. If needed, I can run  `mage protoc`.
can you please tell me, how it needs to be run.

Reply:
We should use the same version unless we explicitly change the protoc version.

Reply:
fixed , please review.

Reply:
Thanks